### ISSUE TITLE
[Dropdown] Fails to initialize if no click trigger is found

### ISSUE THREAD

**Issue by gakimball** [zurb/foundation-sites#7600]
On init, dropdowns look for elements `[data-toggle="id-of-dropdown"]` and use that for `aria-describedby`. However, if no click trigger is found, the plugin fails to initialize because it can't set `aria-describedby`. Most likely that part should just fail silently and keep going.

**Comment by zurbchris** [comment]
After further review, I'm of the opinion that this should stay the way it is, for a couple reasons. The aria support is necessary for accessibility reasons, *and*, there needs to be an anchor point for positioning purposes. Otherwise, there's just a dropdown-pane floating aimlessly.
